applications/peripheral_sensor_node/README.rst

```diff
@@ -0,0 +1,337 @@
+
+Configuration
+*************
+
+The nRF Peripheral Sensor node application is modular and event-driven.
+You can enable and configure the modules separately for the selected board and build type.
+See the documentation page of the selected module for information about functionalities provided by the module and its configuration.
+
+Configuration options
+=====================
+
+Check and configure the following Kconfig options:
+
+.. _CONFIG_APP_CONTROL_BURST_INTERVAL_S:
+
+CONFIG_APP_CONTROL_BURST_INTERVAL_S
+   The application configuration specifies the time in seconds between burst data acquisitions from sensors.
+   If sensor data burst is not triggered by ``wake_up_event`` from IMU, burst is performed every interval value.
+
+.. _CONFIG_APP_CONTROL_BURST_MEAS_CNT:
+
+CONFIG_APP_CONTROL_BURST_MEAS_CNT
+   The application configuration specifies the number of measurements a single burst acquires before going to sleep.
+   The default value is ``10``.
+
+.. _CONFIG_APP_CONTROL_BURST_MEAS_INTERVAL_MS:
+
+CONFIG_APP_CONTROL_BURST_MEAS_INTERVAL_MS
+   The application configuration specifies the time interval between measurements for a single burst.
+   The default value is 1000 ms.
+
+Configuration files
+===================
+
+The nRF Peripheral Sensor node application uses the following files as configuration sources:
+
+* Devicetree Specification (DTS) files - These reflect the hardware configuration.
+  See :ref:`zephyr:dt-guide` for more information about the DTS data structure.
+* Kconfig files - These reflect the software configuration.
+  See :ref:`kconfig_tips_and_tricks` for information about how to configure them.
+* :file:`_def` files - These contain configuration arrays for the application modules.
+  The :file:`_def` files are used by the nRF Peripheral Sensor node application modules and :ref:`lib_caf` modules.
+
+Configuration files are defined in the :file:`applications/peripheral_sensor_node/configuration/nrf54l15dk_nrf54l15_cpuapp@soc1` directory.
+
+Advertising configuration
+-------------------------
+
+If a given build type enables Bluetooth, the :ref:`caf_ble_adv` is used to control the Bluetooth advertising.
+This CAF module relies on :ref:`bt_le_adv_prov_readme` to manage advertising data and to scan response data.
+The nRF Peripheral Sensor node application configures the data providers in the :file:`src/util/Kconfig` file.
+By default, the application enables a set of data providers available in the |NCS|.
+If the Noridc UART Service (NUS) is enabled in the configuration, and the Bluetooth local identity in use has no bond, the application also adds a custom provider that appends UUID128 of NUS to the scan response data.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`applications/peripheral_sensor_node`
+
+The Peripheral Sensor node application is built the same way as any other |NCS| application or sample.
```

Comment:
If it is built the same way as any other |NCS| application or sample, please follow the template: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/application_README.html#building-and-running